rocky/rocky/views/scheduler.py

```diff
@@ -97,13 +97,13 @@ def get_report_schedule_form_start_date_choice(self):
         return self.report_schedule_form_start_date_choice(self.request.POST)
 
     def get_report_schedule_form_start_date(self):
```

Comment:
Is this supposed t be on the scheduler view? Its report related right? Maybe this should be on the repor or on a reportView mixin? The same for the following 4 methods.

Reply:
This comes from another PR (https://github.com/minvws/nl-kat-coordination/pull/3597). @Rieven, can you answer this question? 

Besided this, I'm not sure if we need to fix this in this PR. Might be better to create a new one.

Reply:
Yes, I put this in the SchedulerView, but maybe it is better to move it to the BaseReportView